gpt_index/indices/query/knowledge_graph/query.py

```diff
@@ -6,13 +6,18 @@
 from gpt_index.data_structs.data_structs import KG, Node
 from gpt_index.indices.keyword_table.utils import extract_keywords_given_response
 from gpt_index.indices.query.base import BaseGPTIndexQuery
-from gpt_index.indices.query.embedding_utils import SimilarityTracker
+from gpt_index.indices.query.embedding_utils import (
+    SimilarityTracker,
+    get_top_k_embeddings,
+)
 from gpt_index.indices.query.schema import QueryBundle
 from gpt_index.indices.utils import truncate_text
 from gpt_index.prompts.default_prompts import DEFAULT_QUERY_KEYWORD_EXTRACT_TEMPLATE
 from gpt_index.prompts.prompts import QueryKeywordExtractPrompt
 
 DQKET = DEFAULT_QUERY_KEYWORD_EXTRACT_TEMPLATE
+EMBEDDINGS_ONLY_QUERY = "embedding_only"
```

Comment:
might be worth making this an enum (see QueryMode in gpt_index/indices/query/schema.py as an example)

Reply:
also should there be a "no_embeddings" option? (or is that the None option in embedding_mode)

Reply:
Correct, it won't use embeddings when it is set to None